docs/user/alerting/view-alerts.asciidoc

```diff
@@ -81,3 +81,44 @@ In both *{stack-manage-app} > Alerts* and *{rules-ui}*, you can open the action
 To permanently suppress actions for an alert, open the actions menu and select *Mark as untracked*.
 
 To affect the behavior of the rule rather than individual alerts, check out <<controlling-rules>>.
+
+[discrete]
+[[clean-up-alerts]]
+=== Clean up alerts 
```

Comment:
Do we mark things as technical preview in the docs? If so, we should add that here (and in the 9.1 docs too)